### DISCUSSION TITLE
Organ console on Grandorgue not filling touchscreen

### DISCUSSION THREAD

**Discussion by gavin58** [GrandOrgue/grandorgue#1287] [Help] [answered]
How do you resize the picture of the organ console to fit an illyana 54.5cm touch screen? . I have recently changed from running GrandOrgue from a windows 10 laptop to a windows 10 lenovo thinkcentre M900. When I was running Grandorgue from the laptop, there were no problems getting the organ consoles to fill the entire touch screen, which I could do by simply maximising the console window. Having changed over to the thinkcentre the organ consoles are only filling approx one third of the screen, and there doesn't seem to be any way of resizing them. The maximise button does not work, neither does using the mouse to try and enlarge the window. The size they are at present makes them difficult to use successfully.  
There are no other problems with websites including this one filling the entire touch screen. I also run Jorgan and those consols fill the entire screen.
Any help would be greatly appreciated.

**Comment by rousseldenis** [comment, marked as answer]
> How do you resize the picture of the organ console to fit an illyana 54.5cm touch screen? . I have recently changed from running GrandOrgue from a windows 10 laptop to a windows 10 lenovo thinkcentre M900. When I was running Grandorgue from the laptop, there were no problems getting the organ consoles to fill the entire touch screen, which I could do by simply maximising the console window. Having changed over to the thinkcentre the organ consoles are only filling approx one third of the screen, and there doesn't seem to be any way of resizing them. The maximise button does not work, neither does using the mouse to try and enlarge the window. The size they are at present makes them difficult to use successfully. There are no other problems with websites including this one filling the entire touch screen. I also run Jorgan and those consols fill the entire screen. Any help would be greatly appreciated.

@gavin58 What's your GrandOrgue version?

**Reply by gavin58** [reply]
I've got V0.3.1.2330

**Reply by rousseldenis** [reply]
This has been solved some months ago. Try to install last release from https://github.com/GrandOrgue/grandorgue/releases

**Reply by gavin58** [reply]
Thank you I will do

**Reply by gavin58** [reply]
Worked a treat. Everything is now working as it should
